### PR TITLE
feature/194 Rapid Response form and fund

### PR DESCRIPTION
**Pull request by zerolab**
- creates the global categories
- adds a migration for the RR application form and fund type

<del>Not creating a Round as the historic submissions all have different "rounds" (or the concept of rounds, through deadlines)</del><ins>RR have an open ended round</ins>

### CODE REVIEW

File: opentech/apply/funds/management/commands/seed_rapid_response.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+    def create_rapid_response_round(self, fund):
+        User = get_user_model()
+        lead_qs = User.objects.filter(full_name="Lindsay Beck")
+        lead = lead_qs.first() if lead_qs.count() else User.objects.filter(groups__name=STAFF_GROUP_NAME).first()
+
+        round = Round(
+            title="Rapid Response open round",
+            lead=lead,
+            start_date=date(2015, 8, 28),
```
Comment (todd-dembrey) [inline opentech/apply/funds/management/commands/seed_rapid_response.py]: Why this date?

File: opentech/apply/funds/management/commands/seed_rapid_response.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            fund_form = FundForm.objects.create(fund=fund, form=application_form)
+            fund.forms = [fund_form]
+            fund.save()
+
+        return fund
+
+    def create_rapid_response_round(self, fund):
+        User = get_user_model()
+        lead_qs = User.objects.filter(full_name="Lindsay Beck")
+        lead = lead_qs.first() if lead_qs.count() else User.objects.filter(groups__name=STAFF_GROUP_NAME).first()
```
Comment (todd-dembrey) [inline opentech/apply/funds/management/commands/seed_rapid_response.py]:
```
try:
    User.objects.get(full_name="Lindsay Beck")
except User.DoesNotExist:
     User.objects.filter(groups__name=STAFF_GROUP_NAME).first()
```

File: opentech/apply/funds/management/commands/seed_rapid_response.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+from opentech.apply.home.models import ApplyHomePage
+from opentech.apply.users.groups import STAFF_GROUP_NAME
+
+
+class Command(BaseCommand):
+    help = "Pre-seeds the RR application form and fund type."
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        if Round.objects.filter(title='Rapid Response open round').count():
```
Comment (todd-dembrey) [inline opentech/apply/funds/management/commands/seed_rapid_response.py]: Where does this come from, feels a bit flaky.

File: opentech/apply/categories/management/commands/seed_categories.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from opentech.apply.categories.models import Category, Option
+from opentech.apply.categories.categories_seed import CATEGORIES
+
+
+class Command(BaseCommand):
+    help = "Pre-seeds the global application categories."
```
Comment (todd-dembrey) [inline opentech/apply/categories/management/commands/seed_categories.py]: Do you need to run these in order? can you document this?

File: opentech/apply/funds/management/commands/seed_rapid_response.py (hunk 3 above)
Comment (todd-dembrey) [reply opentech/apply/funds/management/commands/seed_rapid_response.py]: is this linked to the comment you have at the top?

File: opentech/apply/funds/management/commands/seed_rapid_response.py (hunk 1 above)
Comment (zerolab) [reply opentech/apply/funds/management/commands/seed_rapid_response.py]: That is the date of the original Rapid Response request type

Comment (zerolab) [reply opentech/apply/funds/management/commands/seed_rapid_response.py]: Commented

File: opentech/apply/funds/management/commands/seed_rapid_response.py [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    def create_rapid_response_round(self, fund):
+        User = get_user_model()
+
+        try:
+            lead = User.objects.get(full_name="Lindsay Beck")
+        except User.DoesNotExist:
+            lead = User.objects.filter(groups__name=STAFF_GROUP_NAME).first()
+
+        round = Round(
+            title="Rapid Response open round",
```
Comment (todd-dembrey) [inline opentech/apply/funds/management/commands/seed_rapid_response.py]: common value 
            `title="Rapid Response open round",`

Comment (zerolab) [reply opentech/apply/funds/management/commands/seed_rapid_response.py]: sorted

**Pull request merged**